lib/phoenix_live_view.ex

```diff
@@ -261,6 +261,11 @@ defmodule Phoenix.LiveView do
 
   It must always return `{:noreply, socket}`, where `:noreply`
   means no additional information is sent to the client.
+
+  > #### Note {: .warning}
+  >
+  > `handle_params` is only allowed on a root LiveView as it takes the
+  > current url of the page as parameter.
```

Comment:
I think this would be the most precise definition:

```suggestion
  > `handle_params` is only allowed on LiveViews mounted at the router,
  > as it takes the current url of the page as parameter.
```